packages/node/src/utils/cosmos.ts

```diff
@@ -189,14 +189,10 @@ async function getBlockByHeight(
 ): Promise<[BlockResponse, BlockResultsResponse]> {
   return Promise.all([
     api.blockInfo(height).catch((e) => {
-      const error = CosmosClient.handleError(e);
-      logger.error(error, `failed to fetch block info ${height}`);
-      throw error;
+      throw CosmosClient.handleError(e);
```

Comment:
What's the reason for these changes? Logging the reason it failed is useful

Reply:
too much logs. For example, I noticed we log rate limit on every single block in a batch (about 100 times by default)